contents/handbook/company/structure.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Small Teams
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+PostHog is structured for speed, autonomy and innovation.
+
+Many traditional organizations have big, separate functions. You have a product team, an engineering team, customer support, and so on. This slows things down when you scale because there are more layers of communication and complex approval chains. This stifles innovation - you have to get your boss to talk to someone else's boss to get work done. It also means that people can't really see the impact of their work.
+
+PostHog started off as a completely flat company with one big goal: to increase the number of successful products in the world.
+
+As we are getting bigger, we anticipate that it will get harder for people to see the direct impact of their work, which reduces the sense of ownership.
+
+We have therefore introduced Small Teams. These are designed to each operate like a startup. 
+
+# How it works
+
+* The overall goal is for a Small Team to be as close to its own startup as possible, with some centralized processes where it doesn't make sense
+* A Small Team should never be more than six people
+* A Small Team has an Accountable Person responsible for its performance - whoever is most appropriate depending on what the team is working on. This does _not_ mean the most senior person on the team.
+* A Small Team must have (1) a customer (internal or external), (2) a mission and (3) metrics
+* There may be certain functions where at our current stage we don't need a Small Team yet.
+* Each Small Team runs its own retrospective + sprint every week for 30 mins.
+* A Small Team has the final call in which of its features get into production and when, within our existing release schedule.
+* A Small Team can setup it
```

Comment:
@jamesefhawkins is something missing here? Also not all the bullets have full stops.

Reply:
resolved, thanks